src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -3,17 +3,37 @@
 import core.basesyntax.Storage;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private static final int SIZE_OF_ARRAY = 10;
+    private final K[] keys = (K[]) new Object[SIZE_OF_ARRAY];
+    private final V[] values = (V[]) new Object[SIZE_OF_ARRAY];
+    private int elementsAmount;
+    private int valuesCount;
```

Comment:
```suggestion
    private int size;
```
no need for `valuesCount`

Reply:
removed valuesCount